javascript/ql/src/Security/CWE-942/CorsPermissiveConfiguration.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name overly CORS configuration
+ * @description Misconfiguration of CORS HTTP headers allows CSRF attacks.
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity 7.5
+ * @precision high
+ * @id js/cors-misconfiguration
+ * @tags security
+ *       external/cwe/cwe-942
+ */
+
+import javascript
+import semmle.javascript.security.dataflow.CorsPermissiveConfigurationQuery
+import DataFlow::PathGraph
+
+from Configuration cfg, DataFlow::PathNode source, DataFlow::PathNode sink
+where cfg.hasFlowPath(source, sink)
+select sink.getNode(), source, sink, "$@ misconfiguration due to a $@.", sink.getNode(),
+  "CORS Origin", source.getNode(), "too permissive or user controlled value"
```

Comment:
```suggestion
select sink.getNode(), source, sink, "CORS Origin misconfiguration due to a $@.", source.getNode(), "too permissive or user controlled value"
```

The automated warning is saying that the alert-location is already the sink, so adding a link to the same location as the alert location doesn't really make sense.  
This suggestion removes that link.